community/settings.py

```diff
@@ -37,6 +37,30 @@
     'django_distill',
 ]
 
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'formatters': {
+        'standard': {
+            'format': '%(asctime)s [%(levelname)s] %(name)s: %(message)s'
+        }
+    },
+    'handlers': {
+        'communityHandler': {
+            'class': 'logging.FileHandler',
+            'filename': './_site/community.log',
```

Comment:
Hey, put this inside the static folder.
https://docs.djangoproject.com/en/2.0/howto/static-files/

Reply:
this isnt a pre-existing static file.  It has to go in `_site`